test/recipe/Baked.Test.Recipe.Service.Test/CodingStyle/RoutingRichTransients.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Newtonsoft.Json;
+using System.Net.Http.Json;
+
+namespace Baked.Test.CodingStyle;
+
+public class RoutingRichTransients : TestServiceNfr
+{
+    [TestCase("1")]
+    [TestCase("59dfa608-9fe4-4e77-b448-a65adcfda605")]
+    public async Task Get(string id)
+    {
+        var response = await Client.GetAsync($"rich-transient-with-datas/{id}");
+
+        response.StatusCode.ShouldBe(System.Net.HttpStatusCode.OK);
+        dynamic? actual = JsonConvert.DeserializeObject(await response.Content.ReadAsStringAsync());
+        ((string?)actual?.id).ShouldBe(id);
+    }
+
+    [TestCase("rich-transient-with-datas/1/method")]
+    [TestCase("rich-transient-no-datas/1/method")]
+    public async Task Post(string path)
+    {
+        var response = await Client.PostAsync(path, JsonContent.Create(new { text = "text" }));
+
+        response.StatusCode.ShouldBe(System.Net.HttpStatusCode.OK);
+        dynamic? actual = JsonConvert.DeserializeObject<dynamic>(await response.Content.ReadAsStringAsync());
+        ((string?)actual?.id).ShouldBe("1");
+        ((object?)actual?.data).ShouldDeeplyBe(new { text = "text" });
```

Comment:
you are unnecessarily testing object as json coding style in here. you don't even have to have any parameters for this. even if you do, provide a string or some other primitive. if you require a complex type (i can't imagine why) create a strongly typed record for this.